src/home/pages/Withdraw/Withdraw.tsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from 'react';
+
+import { BoxButton, CheckBox } from '@yourssu/design-system-react';
+import { useNavigate } from 'react-router-dom';
+
+import Logo from '@/assets/soomsil_v2_logo.svg';
+
+import {
+  StyledWrapper,
+  StyledWithdrawContainer,
+  StyledTitleText,
+  StyledSubTitleText,
+  StyledText,
+  StyledButtonText,
+  StyledLeft,
+  StyledListItem,
+} from './Withdraw.style';
+
+export const Withdraw = () => {
+  const [agreed, setAgreed] = useState(false);
+  const [draw, setDraw] = useState(false);
+
+  const handleCheckAgree = () => {
+    setAgreed(!agreed);
+  };
```

Comment:
지금은 코드가 문제없이 동작하지만 추후에 비동기 로직등이 `handleCheckAgree()` 함수에 추가되면 상태 업데이트가 예상치 못하게 이루어질 수 있을 것 같아요!

`handleCheckAgree()`와 같이 이전 상태에 기반하여 상태를 업데이트 해야할 때는 update function을 쓰는게 좋은 습관인 것 같습니다.

[Reach batch state update](https://react.dev/learn/queueing-a-series-of-state-updates)

```suggestion
  const handleCheckAgree = () => {
    setAgreed(prevAgreed => !prevAgreed);
  };
```